CLI/secure-env.env

```diff
@@ -0,0 +1,4 @@
+SECURE_ENV_PROJECT_NAME="secure-env"
+SECURE_ENV_HOST="https://secure-env.poc-innovation.com:8200/"
+SECURE_ENV_PORT=8200
+SECURE_ENV_TOKEN="hvs.NWAqB95aaCepzz7GXVrO43KW"
```

Comment:
:warning: Don't push the .env :warning: 

You can remove it from your commit, and rename it `.env` for a more conventional name